scripts/plugins/storage/sqlite/init.sql

```diff
@@ -918,3 +918,12 @@ INSERT INTO foglamp.schedules ( id, schedule_name, process_name, schedule_type,
                 't',                                   -- exclusive
                 'f'                                   -- disabled
               );
+
+-- Insert into category_children
+INSERT INTO foglamp.category_children (parent, child) VALUES ('South', 'dht11');
+INSERT INTO foglamp.category_children (parent, child) VALUES ('North', 'North_Readings_to_PI');
+INSERT INTO foglamp.category_children (parent, child) VALUES ('North', 'North_Statistics_to_PI');
+INSERT INTO foglamp.category_children (parent, child) VALUES ('North', 'North_Readings_to_HTTP');
+INSERT INTO foglamp.category_children (parent, child) VALUES ('North', 'North Readings to PI');
+INSERT INTO foglamp.category_children (parent, child) VALUES ('North', 'North Statistics to PI');
+INSERT INTO foglamp.category_children (parent, child) VALUES ('North', 'North Readings to OCS');
```

Comment:
This will allow us to see these as child if south and north in config tree too. and not in root=true!  

BTW. we also need upgrade/downgrade sql scripts for these